### PR TITLE
=clt #23215 fail with nicer message if PubSubMediator gets null message

### PR DESCRIPTION
**Pull request by ktoso**
Added better message for when `null` it sent to mediator

### CODE REVIEW

**Comment by ktoso** [comment]
Refs #23215

**Review by johanandren — APPROVED** [review]

File: akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala [hunk 1]
```diff
@@ -400,6 +400,7 @@ object DistributedPubSubMediator {
     def wrapIfNeeded: Any ⇒ Any = {
       case msg: RouterEnvelope ⇒ MediatorRouterEnvelope(msg)
       case msg: Any            ⇒ msg
+      case null                ⇒ throw InvalidMessageException("Message must not be null")
```
Comment (patriknw) [inline akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala]: I'd feel more comfortable with placing the `null` case above `: Any`

**Review by patriknw — DISMISSED** [review]

**Review by ktoso — COMMENTED** [review]
Addressed feedback

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**